recipes/check_shapes/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "check_shapes" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: check_shapes
```

Would use `name` variable here as well

```suggestion
  name: {{ name }}
```